src/ci_ex_states/excited_state_solver.cc

```diff
@@ -0,0 +1,1131 @@
+//            size_t idx = nact_ * p + q;
+//            oei_a[idx] = ints_->oei_a(pp, qq);
+//            oei_b[idx] = ints_->oei_b(pp, qq);
+//        }
+//    }
+
+//    fci_ints->set_restricted_one_body_operator(oei_a, oei_b);
+
+//    // Then build the coupling lists
+//    psi::SharedMatrix final_evecs;
+//    psi::SharedVector final_evals;
+
+//    WFNOperator op(mo_symmetry_, fci_ints);
+//    if (diag_method_ != Dynamic) {
+//        op_.clear_op_s_lists();
+//        op_.clear_tp_s_lists();
+//        op.build_strings(ref);
+//        op.op_s_lists(ref);
+//        op.tp_s_lists(ref);
+//    }
+
+//    // Diagonalize full space
+
+//    SparseCISolver sparse_solver(fci_ints);
+//    sparse_solver.set_parallel(true);
+//    sparse_solver.set_force_diag(options_->get_bool("FORCE_DIAG_METHOD"));
+//    sparse_solver.set_e_convergence(options_->get_double("E_CONVERGENCE"));
+//    sparse_solver.set_maxiter_davidson(options_->get_int("DL_MAXITER"));
+//    sparse_solver.set_spin_project(project_out_spin_contaminants_);
+//    sparse_solver.set_spin_project_full(project_out_spin_contaminants_);
+//    sparse_solver.set_guess_dimension(options_->get_int("DL_GUESS_SIZE"));
+//    sparse_solver.set_num_vecs(options_->get_int("N_GUESS_VEC"));
+//    sparse_solver.set_sigma_method(options_->get_str("SIGMA_BUILD_TYPE"));
+//    sparse_solver.set_max_memory(options_->get_int("SIGMA_VECTOR_MAX_MEMORY"));
+
+//    sparse_solver.diagonalize_hamiltonian_map(ref, op, final_evals, final_evecs, nroot_,
+//                                              multiplicity_, diag_method_);
+
+//    outfile->Printf("\n\n");
+//    for (int i = 0; i < nroot_; ++i) {
+//        double abs_energy =
+//            final_evals->get(i) + nuclear_repulsion_energy_ + fci_ints->frozen_core_energy();
+//        double exc_energy = pc_hartree2ev * (final_evals->get(i) - final_evals->get(0));
+//        outfile->Printf("\n  * ACI+es Energy Root %3d        = %.12f Eh = %8.4f eV", i,
+//        abs_energy,
+//                        exc_energy);
+//        //    outfile->Printf("\n  * Adaptive-CI Energy Root %3d + EPT2 = %.12f Eh = %8.4f eV", i,
+//        //                    abs_energy + multistate_pt2_energy_correction_[i],
+//        //                    exc_energy +
+//        //                        pc_hartree2ev * (multistate_pt2_energy_correction_[i] -
+//        //                                         multistate_pt2_energy_correction_[0]));
+//        //    	if(options_->get_str("SIZE_CORRECTION") == "DAVIDSON" ){
+//        //        outfile->Printf("\n  * Adaptive-CI Energy Root %3d + D1   =
+//        //        %.12f Eh = %8.4f eV",i,abs_energy + davidson[i],
+//        //                exc_energy + pc_hartree2ev * (davidson[i] -
+//        //                davidson[0]));
+//        //    	}
+//    }
+
+//    print_wfn(ref, op, final_evecs, nroot_);
+//    max_rdm_level_ = 1;
+//    compute_rdms(fci_ints, ref, op, final_evecs, 0, 0);
+//}
```

Comment:
Again, we can delete these commented lines. A lot of this code is in mrci.cc, where it should be